templates/public-sector/index.html

```diff
@@ -366,53 +364,67 @@ <h3>UAB accelerates research with Canonical and Dell</h3>
     "href": "https://ubuntu.com/aws",
     "text": "Learn more about Canonical's partnership with AWS&nbsp;&rsaquo;",
     "label": "AWS",
-    "image_html": '<img class="p-image-container__image"
-       src="https://assets.ubuntu.com/v1/f75b6e08-aws-logo.png"
-       alt="AWS logo"
-       width="234"
-       height="481" />'
-    },
-    {
-    "href": "https://ubuntu.com/azure",
-    "text": "Learn more about Canonical's partnership with Microsoft Azure&nbsp;&rsaquo;",
-    "label": "Azure",
-    "image_html": '<img class="p-image-container__image"
-       src="https://assets.ubuntu.com/v1/fa0fde10-microsoft-azure-logo.png"
-       alt="Azure logo"
-       width="418"
-       height="481" />'
-    },
-    {
-    "href": "https://ubuntu.com/gcp",
-    "text": "Learn more about Canonical's partnership with Google Cloud&nbsp;&rsaquo;",
-    "label": "GCP",
-    "image_html": '<img class="p-image-container__image"
-       src="https://assets.ubuntu.com/v1/0ad3b97c-google-cloud-logo.png"
-       alt="GCP logo"
-       width="586"
-       height="481" />'
-    },
-    {
-    "href": "https://ubuntu.com/cloud/public-cloud",
-    "text": "Learn more about Canonical's public cloud partnerships&nbsp;&rsaquo;",
-    "label": "IBM Cloud",
-    "image_html": '<img class="p-image-container__image"
-       src="https://assets.ubuntu.com/v1/74e883de-ibm-cloud-logo.png"
-       alt="IBM Cloud logo"
-       width="450"
-       height="481" />'
-    },
-    {
-    "href": "https://ubuntu.com/cloud/public-cloud",
-    "text": "Learn more about Canonical's public cloud partnerships&nbsp;&rsaquo;",
-    "label": "Oracle Cloud",
-    "image_html": '<img class="p-image-container__image"
-       src="https://assets.ubuntu.com/v1/136a94e7-oracle-cloud-logo.png"
-       alt="Oracle Cloud logo"
-       width="588"
-       height="478" />'
-    }
-    ],)
+    "image_html": image(url="https://assets.ubuntu.com/v1/f75b6e08-aws-logo.png",
+    alt="AWS logo",
+    width="234",
+    height="481",
+    hi_def=True,
+    loading="lazy",
+    attrs={"class": "p-image-container__image"}) 
+  },
+  {
+  "href": "https://ubuntu.com/azure",
+  "text": "Learn more about Canonical's partnership with Microsoft Azure&nbsp;&rsaquo;",
+  "label": "Azure",
+  "image_html": image(url="https://assets.ubuntu.com/v1/fa0fde10-microsoft-azure-logo.png",
+  alt="Azure logo",
```

Comment:
```suggestion
  alt="Azure",
```
Could you please remove "logo" from all the alt text?
It is recommended to just use company name.

Reply:
Thanks for spotting.
Removed all